docs/reference/field-properties/README.md

```diff
@@ -14,6 +14,7 @@ Each [Schema Field Type](/reference/field-types) in Apostrophe can take a number
 |help | `string` | | Help text for the field that will appear with the field's label | universal | |
 |htmlHelp | `string` | | Help text with support for HTML markup | universal | |
 |contextual | `boolean` | false | If true, it will prevent the field from appearing in the dialog box for a widget | universal | |
+|permission | `string` | | Specify [the permission level needed](../../core-concepts/users-and-permissions/managing-access-control.md) to edit this field | universal | |
```

Comment:
`/core-concepts` is a top level section, so you can remove the `../../` at the beginning. So, it should simply be `/core-concepts/users-and-permissions/managing-access-control.md`.

Reply:
Done